index.html

```diff
@@ -2653,7 +2653,7 @@ <h3 id="applying">
               </li>
             </ol>
           </li>
-          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
+          <li><a>Trim</a>(<var>value</var>) and convert <a>to ASCII lowercase</a></li>
```

Comment:
I am not sure, but I filed a bug for this. Feel free to add more info to the bug.

I think for this patch you have done what is required and that part is fine. We can fix all the references in another PR when we decide what to do